pinboard/pinboard.py

```diff
@@ -55,8 +55,8 @@ def save(self, update_time=False):
         params = {
             'url': self.url,
             'description': self.description.encode("utf-8"),
-            'extended': self.extended,
-            'tags': self.tags,
+            'extended': self.extended.encode("utf-8"),
+            'tags': [t.encode("utf-8") for t in self.tags],
```

Comment:
Looks great, only thing I would change is the name of the variable in the list comprehension. `t` -> `tag`.